crates/cargo-codspeed/src/build.rs

```diff
@@ -226,7 +226,9 @@ See `cargo codspeed build --help` for more information.");
         ];
 
         // Add the codspeed cfg flag if simulation mode is enabled
```

Comment:
Update comment, or create a function that takes measurement_mode and outputs a bool, named `should_build_benchmark_target_to_run_only_once` or something like this, to convey the intention